master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -99,6 +99,18 @@ spec:
     spec:
       hostNetwork: true
       serviceAccountName: calico-node
+      # Normally, terminationGracePeriodSeconds controls the delay between sending a SIGTERM
```

Comment:
I do appreciate the detail of the comment here - wondering if there is a k8s doc we could link to that contains this information instead?

If not that's fine and we can leave it as-is, but generally prefer to keep the comments short and sweet in these manifests, and this makes it feel like this setting is vastly more important than the others.

Reply:
Agreed, think the new text just about covers it (and the commit message spells it out in detail).  Definitely subtle enough that I was worried I'd forget why I put that there!